R/asis-nodes.R

```diff
@@ -260,3 +260,38 @@ protect_tickbox <- function(body, ns) {
   }
   copy_xml(body)
 }
+
+# FOOTNOTES --------------------------------------------------------------------
+#nocov start
+footnote_check <- function(body, ns = md_ns()) {
```

Comment:
why not `check_footnotes()`?

Reply:
I agree! I had directly copied the `tick_check()` function to make this, and just switched out tick/ticks for footnote/feet :stuck_out_tongue_closed_eyes: